app/controllers/sidebar_sections_controller.rb

```diff
@@ -50,6 +50,17 @@ def update
     render json: failed_json, status: 403
   end
 
+  def reset
+    sidebar_section = SidebarSection.find(params[:id])
```

Comment:
I don't think we rescue from `ActiveRecord::RecordNotFound` errors automatically so it is better to use `#find_by` and raise `Discourse::InvalidParameters` when the `id` param is invalid.